.github/workflows/ci.yml

```diff
@@ -25,3 +28,48 @@ jobs:
       
     - name: gst-inspect
       run: gst-inspect-1.0 build/gst-plugin/libgstdrpai.so
+
+  build-aarch64-yocto-rpm:
+  
+    needs: build
+    runs-on: DC02
+    
+    steps:    
+    - uses: actions/checkout@v3
+      with:
+        repository: MistySOM/rzv2l
+        path: rzv2l
+        ref: master
+        submodules: true
+
+    - uses: actions/checkout@v3
+      with:
+        path: gstreamer1.0-drpai
+
+    - name: Replace the RZV2L gstreamer1.0-drpai with the current one.
+      run: |
+        rm -rf rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai
+        cp -r gstreamer1.0-drpai rzv2l/Build/meta-mistysom/recipes-drpai/
+        echo "SRC_URI = \"file://\${THISDIR}\"" >> rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai/bitbake-recipe.bb
+        echo "S = \"\${WORKDIR}/\${THISDIR}\"" >> rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai/bitbake-recipe.bb
```

Comment:
Why do you write the strings to `bitbake-recipe.bb` just to rename it afterwards to `gstreamer1.0-drpai.bb` ?

Reply:
I can rename it first and then write strings. Does the order matter?

Reply:
no, but why aren't you echo-ing the strings into `gstreamer1.0-drpai.bb` directly? it looks like you're echo-ing into `rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai/bitbake-recipe.bb` jus to rename it afterwards to `rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai/gstreamer1.0-drpai.bb`. so why not echo to  `gstreamer1.0-drpai.bb` to begin with? I feel like I'm missing something, but what is it??? 🤔

Reply:
`bitbake-recipe.bb` has its own contents. I am echoing to the end of it so it would override variables.

Reply:
Yes, I see but it still doesn't make sense to me to do it this way...

Reply:
@mistyron 
How would you suggest this to be implemented?

Reply:
Haha, I can't believe that we're still discussing this. Nevermind, I see now what I've missed before.